pkg/validate/openshiftcluster_validatedynamic.go

```diff
@@ -140,7 +140,7 @@ func (dv *openShiftClusterDynamicValidator) Dynamic(ctx context.Context) error {
 			dv.subscriptionDoc.Subscription.Properties.TenantID,
 			spp.ClientID,
 			string(spp.ClientSecret),
-			nil,
+			dv.env.Environment().ClientSecretCredentialOptions(),
```

Comment:
This is used further down in the same function (158). Can we lift the definition higher up in the function and use it multiple times?

Reply:
No, this is conditioned so that we only create if checkaccess is turned on.

Reply:
Suggestion was applied [here[(https://github.com/Azure/ARO-RP/pull/3335).